solr/bin/solr.cmd

```diff
@@ -383,18 +355,19 @@ goto done
 @echo.
 goto done
 
-
-REM Really basic command-line arg parsing
+REM Parse arguments for special commands (start, stop, restart)
```

Comment:
maybe say "auth" too?

Reply:
This part is skipped in case of the auth command. The auth command has its own parsing plus the java class parsing. If I am not mistaken, this was the same behavior before and if I am not mistaken `bin/solr` does the same. Correct me if I am wrong though.

Reply:
Yeah, it does..  I think the way in `bin/solr` it is that first you do everything.  Then you do auth.   Then you do the remaining ones of start, stop, restart.